src/sites-configuration.ts

```diff
@@ -384,6 +384,13 @@ export const Sites: Record<string, SiteConfiguration> = {
       { ordered: "/tags/{key}={value}" },
       { ordered: "/keys/{key}" }
     ]
+  },
+
+  osmbuildings: {
+    link: "osmbuildings.org",
+    paramOpts: [
+      { ordered: "/?lat={lat}&lon={lon}&zoom={zoom}" }, // &tilt=45&rotation=168
```

Comment:
```suggestion
      urlPattern1, // TODO: &tilt=45&rotation=168
```
for this specific case, there is an optimization

Reply:
I see.